HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -129,7 +129,7 @@ def customiseFor2018Input(process):
 
     return process
 
-def customizeForCSCRecHitDProducer(process):
+def customizeFor33453(process):
```

Comment:
Sorry, I also mistyped it in my suggestion, but the number of the PR is 33543...